src/Components/FeelingRoom/fineAnimations.scss

```diff
@@ -0,0 +1,67 @@
+
+@mixin circle($size, $bc, $z) {
```

Comment:
This mixin is GREAT! Let's continue to use this as we do animations to keep our scss files a little more structured.